grunttasks/copyright.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('copyright', {
+    app: {
+      src: [
+        "{,bin/,config/,db/,scripts/}*.js"
```

Comment:
This was most probably my screwup, but I think these paths are non-kosher for this repo. For example, I only see a docs/*, scripts/*, and test/* folder in the root.

Possibly change to this (since docs/* is only an .md file, and test/* has a special license):
```
"{,scripts/}*.js"
```